.github/workflows/git-release-tag.yaml

```
@@ -14,19 +14,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out source code
-        uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5 # v2
+        uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
         with:
           fetch-depth: 0
       - name: Bump version and push tag
         if: github.ref == 'refs/heads/${{ inputs.branch }}' && github.event_name == 'push'
```

actionlint (1.7.12)

[error] if: condition "github.ref == 'refs/heads/${{ inputs.branch }}' && github.event_name == 'push'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)